Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```
@@ -927,6 +928,12 @@ def main():
             return_results(run_response_play(**args))
         else:
             raise NotImplementedError(f"Command {command} is not implemented")
+    except HTTPError as http_error:
+        if hasattr(http_error, 'response') and hasattr(http_error.response, 'content'):
+            demisto.debug(str(http_error.response.content))
+            return_error(f"Error in API request {str(http_error.response.content)}")
```

It'll be better to raise the error and let main handle the error